seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/parse/MultipleTableJobConfigParser.java

```diff
@@ -165,8 +167,15 @@ public ImmutablePair<List<Action>, Set<URL>> parse() {
             connectorJars.addAll(commonPluginJars);
         }
         ClassLoader parentClassLoader = Thread.currentThread().getContextClassLoader();
-        ClassLoader classLoader =
-                new SeaTunnelChildFirstClassLoader(connectorJars, parentClassLoader);
+
+        ClassLoader classLoader;
+        if (classLoaderService == null) {
+            classLoader = new SeaTunnelChildFirstClassLoader(connectorJars);
```

Comment:
I think we shoud keep this logic.

```suggestion
            classLoader = new SeaTunnelChildFirstClassLoader(connectorJars, parentClassLoader);
```